data-plane/pom.xml

```diff
@@ -28,7 +28,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <java.version>15</java.version>
```

Comment:
there is more, see https://github.com/knative-extensions/eventing-kafka-broker/pull/3516